internal/adapters/cloud/aws/rds/rds.go

```diff
@@ -268,6 +268,7 @@ func (a *adapter) adaptCluster(dbCluster types.DBCluster) (*rds.Cluster, error)
 		LatestRestorableTime: defsecTypes.TimeUnresolvable(dbClusterMetadata),
 		AvailabilityZones:    availabilityZones,
 		DeletionProtection:   defsecTypes.Bool(aws.ToBool(dbCluster.DeletionProtection), dbClusterMetadata),
+		SkipFinalSnapshot:    defsecTypes.Bool(false, dbClusterMetadata),
```

Comment:
I'm implementing this modification to support `adapter-lint` .
The direct assignment of `false` is because there is no `SkipFinalSnapshot` in [the definitio](https://github.com/aws/aws-sdk-go-v2/blob/a81a9f719ac616318333ee18ee257110d11c95ff/service/rds/types/types.go#L432)n here.

If this modification is not appropriate, this pull request will Close 😢

Reply:
I see. That's reasonable, it is strange that the definition does not contain this field.

Reply:
> it is strange that the definition does not contain this field.

I assume that the AWS team can't implement this attribute because it is not a state.
`-skip-final-shanpshot` is a flag in the `delete-db-cluster` command.

https://docs.aws.amazon.com/cli/latest/reference/rds/delete-db-cluster.html#options

However, I can set SkipFinalSnapshot as a attribute in Terraform because they can call the `delete-db-cluster` command when calling terraform destroy.